lib/sqlalchemy/sql/elements.py

```diff
@@ -1503,6 +1517,7 @@ def proxy_set(self) -> FrozenSet[ColumnElement[Any]]:
 
     @util.memoized_property
     def _expanded_proxy_set(self) -> FrozenSet[ColumnElement[Any]]:
+        # type: ignore [no-untyped-call]
         return frozenset(_expand_cloned(self.proxy_set))
```

Comment:
Untyped call to `sql.base:_expand_cloned()`. I think I can see which types it needs, but left it as-is for now